packages/insomnia-inso/src/commands/export-specification.js

```diff
@@ -11,9 +11,9 @@ export type ExportSpecificationOptions = GlobalOptions & {
 
 export async function exportSpecification(
   identifier: ?string,
-  { output, workingDir, appDataDir, ci }: ExportSpecificationOptions,
+  { output, workingDir, appDataDir, ci, src }: ExportSpecificationOptions,
 ): Promise<boolean> {
-  const db = await loadDb({ workingDir, appDataDir, filterTypes: ['ApiSpec'] });
+  const db = await loadDb({ workingDir, appDataDir, filterTypes: ['ApiSpec'], src: src });
```

Comment:
```suggestion
  const db = await loadDb({ workingDir, appDataDir, filterTypes: ['ApiSpec'], src });
```

Could you please make this change against the other instances as well? I won't comment on each one 🙂 I'm sure there's an ES Lint rule for this

Reply:
Fixed it